range_related_properties.md

```diff
@@ -0,0 +1,147 @@
+# Range Related Properties
+
+The range related properties are used to fully present and describe an element whose value can be one of a range of values to assistive technologies. These properties communicate the maximum and minimum values for the element and the element's current value.
+
+The range properties can also be used to fully present and describe an element whose value can be one of a list of non-numeric values. In this scenario, all possible text values of the element will be programmatically mapped to numbers within the numeric range.
```

Comment:
"In this scenario, all possible text values of the element will be programmatically mapped to numbers within the numeric range."

I feel like for this sentence it is not clear who is programmatically doing the mapping -- the answer is that the user who is adding the role should. I'm not sure how to say this -- how do I refer to this user?

Reply:
author